utils/ccloud_library.sh

```diff
@@ -522,30 +522,37 @@ function ccloud::validate_ccloud_config() {
 }
 
 function ccloud::validate_ksqldb_up() {
-  ksqldb_endpoint=$1
-  ccloud_config_file=$2
-  credentials=$3
+	[ -z "$1" ] && {
+		echo "ccloud::validate_ksqldb_up expects one parameter (ksqldb endpoint)"
+		exit 1
+	}
+
+	[ $# -gt 1 ] && echo "WARN: ccloud::validate_ksqldb_up function expects one parameter"
+
+  local ksqldb_endpoint=$1
 
   ccloud::validate_logged_in_ccloud_cli || exit 1
 
   if [[ "$ksqldb_endpoint" == "" ]]; then
     echo "ERROR: Provision a ksqlDB cluster via the Confluent Cloud UI and add the configuration parameter ksql.endpoint and ksql.basic.auth.user.info into your Confluent Cloud configuration file at $ccloud_config_file and try again."
```

Comment:
`ccloud_config_file` was removed as an argument, so is the value known in this function?

Reply:
Sorry, can you clarify the question?  My idea with the warning is just to provide an indiciation of improper invocation of the function.

Reply:
I understand now, sorry the GitHub UI confused me with which line you were indicating.  Thanks for catching this, will address

Reply:
removed in latest